build/multi-private/Dockerfile.multi

```diff
@@ -29,7 +29,6 @@ RUN if [[ $(echo $TARGETPLATFORM | cut -f2 -d '/') == "amd64" ]]; then curl http
 RUN if [[ $(echo $TARGETPLATFORM | cut -f2 -d '/') == "amd64" ]]; then curl http://ack-csiplugin.oss-cn-hangzhou.aliyuncs.com/pre/ossfs/ossfs_${update_ossfsVer}_centos7.0_x86_64.rpm -o /root/ossfs_${update_ossfsVer}_centos7.0_x86_64.rpm; fi
 RUN if [[ $(echo $TARGETPLATFORM | cut -f2 -d '/') == "amd64" ]]; then curl http://ack-csiplugin.oss-cn-hangzhou.aliyuncs.com/pre/ossfs/ossfs_${update_ossfsVer8u}_centos8_x86_64.rpm -o /root/ossfs_${update_ossfsVer8u}_centos8_x86_64.rpm; fi
 
-RUN if [[ $(echo $TARGETPLATFORM | cut -f2 -d '/') == "amd64" ]]; then mv /multi_data/jindofs-fuse-3.7.3-20211207.tar.gz /; tar zvxf /jindofs-fuse-3.7.3-20211207.tar.gz -C /; fi
```

Comment:
remove the jindofs-fuse-3.7.3-20211207.tar.gz file from git repo together with this line?

Reply:
done